internal/provider/common/bootstrap.go

```diff
@@ -375,26 +375,9 @@ func startInstanceZones(env environs.Environ, ctx envcontext.ProviderCallContext
 // This is all that is required for the bootstrap to continue. Further configured
```

Comment:
Line above this is now out of date given the code no longer opens the api port.
However, it seems removing the api port breaks bootstrap on gce - the bootstrap cli cannot connect

Reply:
Worked this time. Must have been an ssh issue